### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by anero**
Typos and content fixes

### CODE REVIEW

File: source/includes/_bulk_data.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ The core steps to get this up and running are:
 4. Setting up your `aws-lambda-redshift-loader` lambda.
 5. Configure your bulk data webhook in the Control Shift settings console.
 
-**Note:** All resources should be set up in the same AWS region (ex: us-east-1) so they can access each other. "US Standard" and "us-east-1" are the same thing.
+**Note:** All resources should be set up in the same AWS region (i.e.: us-east-1) so they can access each other. "US Standard" and "us-east-1" are the same thing.
```
Comment (lavaturtle) [inline source/includes/_bulk_data.md]: should use "e.g.", not "i.e.", here.

(Latin Nerd sidebar: "i.e." == "id est" ~= "that is", while "e.g." == "exempli gratia" ~= "for example")

Comment (lavaturtle) [inline source/includes/_bulk_data.md]: well, unless it actually needs to specifically be us-east-1

Comment (anero) [inline source/includes/_bulk_data.md]: Nope, it's just an example

File: source/includes/_bulk_data.md [hunk 2]
```diff
@@ -310,17 +310,18 @@ We need to allow our user to access our aws-lambda-redshift-loader to access our
...
-4. Set the Runtime to Node.js 0.10.
-5. Use the default values for the handler, and in the Role drop-down, select "* Basic Execution Role." A IAM creation wizard will open in a new window.
-6. Follow the wizard, selecting to "Create a new IAM Role" and name it as you like.  Click to "View the Policy" and then click edit.
-7. Copy and paste the the AWS Lambda Execution Role permissions from the example "AWS Lambda Execution Role" or from the official [readme](https://github.com/awslabs/aws-lambda-redshift-loader#getting-started---lambda-execution-role).
-8. Then click "Add Policy." Select the `AmazonRedshiftFullAccess` role and add it, then add another role - `AmazonDMSRedshiftS3Role`.
-9. Navigate back to your Lambda setup tab and set the max timeout (5 minutes) to accommodate potentially long COPY times.
-10. From the VPC dropdown, select your default VPC.
-11. Leave the rest of the settings alone.
-12. Click next and then click to create your Lambda.
+2. Select Create a Lambda function and enter a name (i.e.: controlShiftRedshiftLoader).
```
Comment (lavaturtle) [inline source/includes/_bulk_data.md]: i.e. -> e.g. here too

File: source/includes/_webhooks.md.erb [hunk 3]
```diff
@@ -8,9 +8,9 @@ We recommend using a tool like [RequestBin](http://requestb.in/) to help debug a
 
 If you need additional information about webhooks or how to use them, please send us a support email [support@controlshiftlabs.com](mailto:support@controlshiftlabs.com). We'd also love stories about how customers are using webhooks.
 
-### Identifying Unique Webhook Events
+### Retries
 
-In the extermely unlikely event that a webhook is double set for some reason (error on our end, Internet gremlins, etc.), you can always identify webhooks by their unique `jid` (job id). If an incoming webhook has the same `jid` as a webhook that has already been processed, you can safely ignore the new one.
+Webhook notifications successful processed expect the response to have an HTTP 2XX status code. Responses returning a different status code will be marked as failed and automatically retried up to 9 times with an exponential backoff (i.e.: after 15s, 30s, 90s, etc.).
```
Comment (woodhull) [inline source/includes/_webhooks.md.erb]: should include a note about how, if we exhaust retries, we'll disable webhook, and you'll need to manually re-enable it again through web ui.

Comment (anero) [inline source/includes/_webhooks.md.erb]: Agree, that's a good idea

Comment (lavaturtle) [inline source/includes/_webhooks.md.erb]: The wording of the first part of this first sentence seems odd/unclear.  Can it be reworded?

**Comment by lavaturtle** [comment]
:+1:

**Pull request merged**